spec/controllers/presentations_controller_spec.rb

```diff
@@ -55,29 +55,12 @@
 
     before { mock_pundit_user_as(non_admin) }
 
-    describe 'GET #new' do
```

Comment:
Yeah the reason I took these tests out was with the presentation pundit policy changing, non-admin and admin users share the same privileges as each other when accessing the "new" and "create" controller actions. So if I were to keep these tests here I would just have to change them to be testing the exact same thing that the admin tests are testing.

Feel like if the policy is tested, which it is. We can be sure that this is working as intended.